javascript/ql/test/experimental/PoI/CommandInjectionPoIConfiguration.ql

```diff
@@ -0,0 +1,15 @@
+/**
+ * @kind problem
+ */
+
+import javascript
+import experimental.PoI.PoI::PoI
```

Comment:
`PoI.PoI::PoI` is a bit much perhaps.

Could we change it to `import experimental.poi.PoI`?
- Folder names are usually lower-case, when they are to be used in imports.
- The inner module named `PoI` seems unnecessary given that this module is not exported from `javascript.qll`.
- A module named the same as its file is an anti-pattern as it leads to compiler warnings when a file in the same folder imports `PoI`.

When/if it moves to the core libraries, we could use a path like `semmle.javascript.explore.PoI`.